packages/app/src/server/models/user.js

```diff
@@ -47,7 +47,7 @@ module.exports = function(crowi) {
     name: { type: String },
     username: { type: String, required: true, unique: true },
     email: { type: String, unique: true, sparse: true },
-    slackMemberId: { type: String, unique: true },
+    slackMemberId: { type: String, unique: true, sparse: true },
```

Comment:
## Reference
https://stackoverflow.com/questions/24430220/e11000-duplicate-key-error-index-in-mongodb-mongoose